graphql/types/ReturnRequest.graphql

```diff
@@ -136,7 +136,7 @@ type ReturnRequestItem {
   imageUrl: String!
   unitMultiplier: Float!
   sellerId: String!
-  sellerName: String!
+  sellerName: String
```

Comment:
Since we are resolving it before sending the response, we can keep it as non-nullable.

It's important to note that when you change something on GraphQL (also routes on node), depending on the changes we get a warning in the terminal like the below one. If we proceed with these change, we would have to release a new major.
![Screen Shot 2022-08-30 at 6 21 18 PM](https://user-images.githubusercontent.com/38737958/187545827-ab0699d2-aa07-4888-ba36-3c60fb1ffc87.png)